safe_web_api_playground/static/js/code-snippets/safeMutableDataEntries.js

```diff
@@ -22,20 +22,16 @@ module.exports = {
       return `Value: ${value.buf}`;
     },
 
-    forEach: async () => {
+    listEntries: async () => {
       // Use safeMutableData.getEntries to obtain entriesHandle
 
-      // Look for console log output in your safe browser console, not this console
       try {
-        await window.safeMutableDataEntries.forEach(entriesHandle, (key, value) => {
-
-          console.log(key + ': ' + value.buf);
-
-        })
+        var entriesArray = await window.safeMutableDataEntries.listEntries(entriesHandle);
+	console.log(entriesArray);
```

Comment:
Perhaps we can still do a forEach on the array and log the values as before so people understand the object we are returning for each entry?